coordinator/migrations/2023-07-13-010300_trade_fees/up.sql

```diff
@@ -0,0 +1,26 @@
+-- Your SQL goes here
+-- insert a payment with amount zero that is a dummy default payment that corresponds to this actual zero-amount invoice:
+-- lnbcrt0m1pj26wczdq50fjhymeqd9h8vmmfvdjsnp4qtwk40kf07d8fzlhdt2s9vqyeczarvk37safua4a0kz7wellkq3vjpp5d7dce9wzhfa3s4a3n7t4xu3ss9slah6sl6mc5ffzqqf45sv82ggqsp54cmyvxklz48ap60guqsln4wuh2ranlap3grg2djt5ykz7005fc6s9qyysgqcqpcxq8z7ps9nqepcz2l2s4qrp6qeks68qry55ylcz542c8a6m6fffmhlmhtn9nnf458ngv83pk0473xfecdk7m7maumqk4jvaymdg7fpgn9tujcehxpqqte97xd
+-- We insert this dummy payment so we are able to sum up fees on already existing trades and make the fee payment hash a mandatory field.
```

Comment:
🙃 It's also for closing expired positions, at least at the moment.

Reply:
Yeah, that was not planned initially - I can adapt the comment or even opt for having *another* dummy value in the db for that. 
I thought about changing the model make the fee optional (removing the `NOT NULL`) constraint instead, but I feel eventually there *should* be a fee for every trade. Or not...? CC @bonomat :)

Reply:
I'm not sure I fully understand the question: are you asking if there should be a fee for every trade? Imho the answer to this question is yes, if it is a trade, i.e. it remove liquidity from the orderbook, then there should be a fee involved.